<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mapImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineGetX">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineGetY">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbShow">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>show</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="radioLayout">
     <item>
      <widget class="QRadioButton" name="radioScheme">
       <property name="text">
        <string>scheme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioSatelite">
       <property name="text">
        <string>satellite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAll">
       <property name="text">
        <string>all</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineAddress">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>640</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelWA">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>640</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>write address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>640</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAnnul">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>640</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>annul</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelForInfo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>670</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>620</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>write post index</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sbScale">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>530</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelScale">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>scale:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
